src/openms/include/OpenMS/MATH/STATISTICS/LinearRegression.h

```diff
@@ -166,6 +164,12 @@ namespace OpenMS
       /// Non-mutable access to relative standard deviation
       double getRSD() const;
 
+      /// given x compute y = slope * x + intercept
+      static inline double computePointY(double x, double slope, double intercept)
```

Comment:
bit of a strange name
eval or just Y would probably be better

Reply:
I just moved the function. I can check how widely it is used and maybe rename.